src/app/globals.css

```diff
@@ -40,4 +40,24 @@ body {
   width: 120px;
   height: 120px;
   animation: spin 2s linear infinite;
+}
+
+::-webkit-scrollbar {
+  width: 6px;
+}
+
+/* Track */
+::-webkit-scrollbar-track {
+  background: #f1f1f1;
+}
+
+/* Handle */
+::-webkit-scrollbar-thumb {
+  background: #888;
+  border-radius: 5px;
+}
+
+/* Handle on hover */
+::-webkit-scrollbar-thumb:hover {
+  background: #555;
 }
```

Comment:
![image](https://github.com/yenting-biao/FoodDate/assets/91837449/a069ec68-0642-409f-b4a4-02de5529f85b)

Note that these are for prettier scroll bar